pkg/config/config_template.yaml

```
@@ -474,6 +474,12 @@ api_key:
   #
   # receiver_port: 8126
 
+  ## @param receiver_socket - string - optional
+  ## Collect your traces through a Unix Domain Sockets.
+  ## It is off by default. When set, it must point to a valid socket file.
+  #
+  # receiver_socket: <APM_SOCKET>
```

I would use a real example here. `<APM_SOCKET>` doesn't really mean anything. 
```suggestion
  # receiver_socket: /path/to/unix.sock
```

---

What do you think of this: to keep the placeholder layout ?
```suggestion
  # receiver_socket: <UNIX_SOCKET_PATH>.sock
```

---

Sure, but let's leave out the extension, I think it isn't mandatory. So if you prefer to use that layout just do:
```suggestion
  # receiver_socket: <UNIX_SOCKET_PATH>
```